react/components/ProductImages/components/Carousel/ThumbnailSwiper.js

```diff
@@ -77,13 +77,12 @@ const ThumbnailSwiper = ({
     'db-ns': hasThumbs,
     mt3: !isThumbsVertical,
     'w-20 bottom-0 top-0 absolute': isThumbsVertical,
-    'left-0 pr5':
+    'left-0':
```

Comment:
Why those classes were removed? Won't those changes affect the layout?

Reply:
foram colocadas em outro lugar

Reply:
they were moved elsewhere to prevent the image from moving when the thumbnail would be rendered